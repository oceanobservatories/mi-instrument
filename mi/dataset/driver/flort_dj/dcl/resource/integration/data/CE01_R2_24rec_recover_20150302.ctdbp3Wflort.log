2015/03/02 00:28:10.283 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/03/02 00:30:16.452 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/03/02 00:30:23.389 [ctdbp3:DLOGP6]: 11.9151,  3.65713,    1.087, 0, 0, 0, 02 Mar 2015 00:30:16
2015/03/02 00:30:23.493 [ctdbp3:DLOGP6]:S>
2015/03/02 00:30:23.504 [ctdbp3:DLOGP6]:!!! Low Battery  !!!
2015/03/02 00:31:04.552 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/03/02 01:28:21.056 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/03/02 01:30:17.224 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/03/02 01:30:24.160 [ctdbp3:DLOGP6]: 11.8921,  3.66142,    1.106, 0, 0, 0, 02 Mar 2015 01:30:16
2015/03/02 01:30:24.264 [ctdbp3:DLOGP6]:S>
2015/03/02 01:30:24.276 [ctdbp3:DLOGP6]:!!! Low Battery  !!!
2015/03/02 01:31:05.304 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/03/02 02:28:11.914 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/03/02 02:30:16.146 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/03/02 02:30:23.083 [ctdbp3:DLOGP6]: 11.8763,  3.66721,    1.156, 0, 0, 0, 02 Mar 2015 02:30:15
2015/03/02 02:30:23.194 [ctdbp3:DLOGP6]:S>
2015/03/02 02:30:23.204 [ctdbp3:DLOGP6]:!!! Low Battery  !!!
2015/03/02 02:31:06.266 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/03/02 03:28:08.816 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/03/02 03:30:15.057 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/03/02 03:30:21.994 [ctdbp3:DLOGP6]: 11.8734,  3.67609,    1.043, 0, 0, 0, 02 Mar 2015 03:30:14
2015/03/02 03:30:22.099 [ctdbp3:DLOGP6]:S>
2015/03/02 03:30:22.109 [ctdbp3:DLOGP6]:!!! Low Battery  !!!
2015/03/02 03:31:05.147 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/03/02 04:28:11.501 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/03/02 04:30:17.675 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/03/02 04:30:24.620 [ctdbp3:DLOGP6]: 11.8662,  3.67795,    1.073, 0, 0, 0, 02 Mar 2015 04:30:17
2015/03/02 04:30:24.718 [ctdbp3:DLOGP6]:S>
2015/03/02 04:30:24.729 [ctdbp3:DLOGP6]:!!! Low Battery  !!!
2015/03/02 04:31:03.766 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/03/02 05:28:09.238 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/03/02 05:30:15.369 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/03/02 05:30:22.306 [ctdbp3:DLOGP6]: 11.8806,  3.68124,    1.055, 0, 0, 0, 02 Mar 2015 05:30:15
2015/03/02 05:30:22.410 [ctdbp3:DLOGP6]:S>
2015/03/02 05:30:22.421 [ctdbp3:DLOGP6]:!!! Low Battery  !!!
2015/03/02 05:31:03.550 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/03/02 06:28:11.311 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/03/02 06:30:17.444 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/03/02 06:30:24.381 [ctdbp3:DLOGP6]: 11.8649,  3.67910,    1.092, 0, 0, 0, 02 Mar 2015 06:30:17
2015/03/02 06:30:24.485 [ctdbp3:DLOGP6]:S>
2015/03/02 06:30:24.496 [ctdbp3:DLOGP6]:!!! Low Battery  !!!
2015/03/02 06:31:03.535 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/03/02 07:28:07.695 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/03/02 07:30:15.825 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/03/02 07:30:22.763 [ctdbp3:DLOGP6]: 11.9226,  3.69672,    1.068, 0, 0, 0, 02 Mar 2015 07:30:15
2015/03/02 07:30:22.866 [ctdbp3:DLOGP6]:S>
2015/03/02 07:30:22.878 [ctdbp3:DLOGP6]:!!! Low Battery  !!!
2015/03/02 07:31:03.905 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/03/02 08:28:10.091 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/03/02 08:30:16.223 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/03/02 08:30:23.158 [ctdbp3:DLOGP6]: 11.9075,  3.69991,    1.084, 0, 0, 0, 02 Mar 2015 08:30:15
2015/03/02 08:30:23.262 [ctdbp3:DLOGP6]:S>
2015/03/02 08:30:23.274 [ctdbp3:DLOGP6]:!!! Low Battery  !!!
2015/03/02 08:31:04.312 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/03/02 09:28:08.877 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/03/02 09:30:14.999 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/03/02 09:30:21.954 [ctdbp3:DLOGP6]: 11.8547,  3.69266,    1.081, 0, 0, 0, 02 Mar 2015 09:30:14
2015/03/02 09:30:22.106 [ctdbp3:DLOGP6]:S>
2015/03/02 09:30:22.107 [ctdbp3:DLOGP6]:!!! Low Battery  !!!
2015/03/02 09:31:05.138 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/03/02 10:28:07.221 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/03/02 10:30:16.267 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/03/02 10:30:23.204 [ctdbp3:DLOGP6]: 11.7898,  3.68496,    1.069, 0, 0, 0, 02 Mar 2015 10:30:16
2015/03/02 10:30:23.308 [ctdbp3:DLOGP6]:S>
2015/03/02 10:30:23.319 [ctdbp3:DLOGP6]:!!! Low Battery  !!!
2015/03/02 10:31:13.317 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/03/02 11:28:07.227 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/03/02 11:30:16.294 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/03/02 11:30:23.230 [ctdbp3:DLOGP6]: 11.7882,  3.68359,    1.037, 0, 0, 0, 02 Mar 2015 11:30:16
2015/03/02 11:30:23.334 [ctdbp3:DLOGP6]:S>
2015/03/02 11:30:23.345 [ctdbp3:DLOGP6]:!!! Low Battery  !!!
2015/03/02 11:31:06.466 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/03/02 12:28:08.775 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/03/02 12:30:15.919 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/03/02 12:30:22.856 [ctdbp3:DLOGP6]: 11.7010,  3.67385,    1.026, 0, 0, 0, 02 Mar 2015 12:30:15
2015/03/02 12:30:22.961 [ctdbp3:DLOGP6]:S>
2015/03/02 12:30:22.972 [ctdbp3:DLOGP6]:!!! Low Battery  !!!
2015/03/02 12:31:06.002 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/03/02 13:28:09.109 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/03/02 13:30:16.238 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/03/02 13:30:23.170 [ctdbp3:DLOGP6]: 11.6997,  3.67370,    1.031, 0, 0, 0, 02 Mar 2015 13:30:15
2015/03/02 13:30:23.274 [ctdbp3:DLOGP6]:S>
2015/03/02 13:30:23.286 [ctdbp3:DLOGP6]:!!! Low Battery  !!!
2015/03/02 13:31:06.400 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/03/02 14:28:09.376 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/03/02 14:30:16.444 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/03/02 14:30:23.382 [ctdbp3:DLOGP6]: 11.7130,  3.67786,    0.989, 1001, 185, 76, 02 Mar 2015 14:30:16
2015/03/02 14:31:05.517 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/03/02 15:28:08.222 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/03/02 15:30:15.344 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/03/02 15:30:22.281 [ctdbp3:DLOGP6]: 11.7264,  3.68068,    1.025, 0, 0, 0, 02 Mar 2015 15:30:15
2015/03/02 15:30:22.386 [ctdbp3:DLOGP6]:S>
2015/03/02 15:30:22.397 [ctdbp3:DLOGP6]:!!! Low Battery  !!!
2015/03/02 15:31:05.427 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/03/02 16:28:06.997 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/03/02 16:30:02.912 [ctdbp3:DLOGP6]:SBE 16plus
2015/03/02 16:30:15.965 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/03/02 16:30:22.909 [ctdbp3:DLOGP6]: 11.7682,  3.68204,    1.005, 724, 158, 102, 02 Mar 2015 16:30:15
2015/03/02 16:31:02.928 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/03/02 17:28:08.147 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/03/02 17:30:15.245 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/03/02 17:30:22.183 [ctdbp3:DLOGP6]: 11.7858,  3.68081,    1.095, 0, 0, 0, 02 Mar 2015 17:30:14
2015/03/02 17:30:22.287 [ctdbp3:DLOGP6]:S>
2015/03/02 17:30:22.298 [ctdbp3:DLOGP6]:!!! Low Battery  !!!
2015/03/02 17:31:05.328 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/03/02 18:28:07.576 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/03/02 18:30:16.629 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/03/02 18:30:23.568 [ctdbp3:DLOGP6]: 11.8341,  3.68205,    1.057, 0, 0, 0, 02 Mar 2015 18:30:16
2015/03/02 18:30:23.676 [ctdbp3:DLOGP6]:S>
2015/03/02 18:30:23.684 [ctdbp3:DLOGP6]:!!! Low Battery  !!!
2015/03/02 18:31:04.713 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/03/02 19:28:08.222 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/03/02 19:30:15.362 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/03/02 19:30:22.299 [ctdbp3:DLOGP6]: 11.8942,  3.67996,    1.020, 0, 0, 0, 02 Mar 2015 19:30:15
2015/03/02 19:30:22.403 [ctdbp3:DLOGP6]:S>
2015/03/02 19:30:22.414 [ctdbp3:DLOGP6]:!!! Low Battery  !!!
2015/03/02 19:31:05.514 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/03/02 20:28:09.095 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/03/02 20:30:16.172 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/03/02 20:30:23.108 [ctdbp3:DLOGP6]: 11.9130,  3.67800,    1.088, 0, 0, 0, 02 Mar 2015 20:30:15
2015/03/02 20:30:23.212 [ctdbp3:DLOGP6]:S>
2015/03/02 20:30:23.224 [ctdbp3:DLOGP6]:!!! Low Battery  !!!
2015/03/02 20:31:04.275 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/03/02 21:28:09.752 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/03/02 21:30:16.899 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/03/02 21:30:23.836 [ctdbp3:DLOGP6]: 11.9373,  3.68169,    1.053, 0, 0, 0, 02 Mar 2015 21:30:16
2015/03/02 21:30:23.940 [ctdbp3:DLOGP6]:S>
2015/03/02 21:30:23.965 [ctdbp3:DLOGP6]:!!! Low Battery  !!!
2015/03/02 21:31:05.032 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/03/02 22:28:08.059 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/03/02 22:30:15.127 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/03/02 22:30:22.064 [ctdbp3:DLOGP6]: 11.9108,  3.68725,    1.064, 0, 0, 0, 02 Mar 2015 22:30:14
2015/03/02 22:30:22.168 [ctdbp3:DLOGP6]:S>
2015/03/02 22:30:22.180 [ctdbp3:DLOGP6]:!!! Low Battery  !!!
2015/03/02 22:31:05.209 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2015/03/02 23:28:08.114 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2015/03/02 23:30:15.224 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2015/03/02 23:30:22.161 [ctdbp3:DLOGP6]: 11.9229,  3.69631,    1.067, 0, 0, 0, 02 Mar 2015 23:30:14
2015/03/02 23:30:22.265 [ctdbp3:DLOGP6]:S>
2015/03/02 23:30:22.276 [ctdbp3:DLOGP6]:!!! Low Battery  !!!
2015/03/02 23:31:05.306 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
