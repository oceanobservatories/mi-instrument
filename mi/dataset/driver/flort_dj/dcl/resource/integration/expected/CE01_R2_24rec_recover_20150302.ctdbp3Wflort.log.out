
 subsite  | node  | sensor       | bin        | method    | time                  | deployment | internal_timestamp | port_timestamp        | preferred_timestamp | quality_flag | date_string | dcl_controller_timestamp | measurement_wavelength_beta | measurement_wavelength_cdom | measurement_wavelength_chl | raw_internal_temp | raw_signal_beta | raw_signal_cdom | raw_signal_chl | time_string
----------+-------+--------------+------------+-----------+-----------------------+------------+--------------------+-----------------------+---------------------+--------------+-------------+--------------------------+-----------------------------+-----------------------------+----------------------------+-------------------+-----------------+-----------------+----------------+-------------
 CE01ISSM | SBD17 | 06-FLORTD000 | 3634243200 | recovered |  3634245023.388999939 |          2 |         3634245016 |  3634245023.388999939 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3634243200 | recovered | 3634248624.1599998474 |          2 |         3634248616 | 3634248624.1599998474 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3634243200 | recovered | 3634252223.0830001831 |          2 |         3634252215 | 3634252223.0830001831 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3634243200 | recovered |  3634255821.993999958 |          2 |         3634255814 |  3634255821.993999958 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3634243200 | recovered | 3634259424.6199998856 |          2 |         3634259417 | 3634259424.6199998856 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3634243200 | recovered | 3634263022.3059997559 |          2 |         3634263015 | 3634263022.3059997559 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3634243200 | recovered |  3634266624.381000042 |          2 |         3634266617 |  3634266624.381000042 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3634243200 | recovered | 3634270222.7630000114 |          2 |         3634270215 | 3634270222.7630000114 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3634243200 | recovered | 3634273823.1579999924 |          2 |         3634273815 | 3634273823.1579999924 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3634243200 | recovered | 3634277421.9539999962 |          2 |         3634277414 | 3634277421.9539999962 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3634243200 | recovered | 3634281023.2039999962 |          2 |         3634281016 | 3634281023.2039999962 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3634243200 | recovered | 3634284623.2300000191 |          2 |         3634284616 | 3634284623.2300000191 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3634243200 | recovered | 3634288222.8559999466 |          2 |         3634288215 | 3634288222.8559999466 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3634243200 | recovered | 3634291823.1700000763 |          2 |         3634291815 | 3634291823.1700000763 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3634243200 | recovered | 3634295423.3819999695 |          2 |         3634295416 | 3634295423.3819999695 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |            1001 |              76 |            185 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3634243200 | recovered | 3634299022.2810001373 |          2 |         3634299015 | 3634299022.2810001373 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3634243200 | recovered | 3634302622.9089999199 |          2 |         3634302615 | 3634302622.9089999199 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |             724 |             102 |            158 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3634243200 | recovered | 3634306222.1830000877 |          2 |         3634306214 | 3634306222.1830000877 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3634243200 | recovered | 3634309823.5679998398 |          2 |         3634309816 | 3634309823.5679998398 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3634243200 | recovered | 3634313422.2989997864 |          2 |         3634313415 | 3634313422.2989997864 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3634243200 | recovered | 3634317023.1079998016 |          2 |         3634317015 | 3634317023.1079998016 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3634243200 | recovered | 3634320623.8359999657 |          2 |         3634320616 | 3634320623.8359999657 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3634243200 | recovered | 3634324222.0640001297 |          2 |         3634324214 | 3634324222.0640001297 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null
 CE01ISSM | SBD17 | 06-FLORTD000 | 3634243200 | recovered | 3634327822.1610002518 |          2 |         3634327814 | 3634327822.1610002518 |      port_timestamp |           ok |        null |                     null |                        null |                        null |                       null |              null |               0 |               0 |              0 |        null

(24 rows)
